core-processor/src/processing.rs

```diff
@@ -119,11 +111,11 @@ where
     // Waking fee: double write cost for removal from waitlist
     // and further enqueueing.
     let msg_ctx_settings = ContextSettings {
-        sending_fee: write_cost.saturating_mul(2),
-        scheduled_sending_fee: write_cost.saturating_mul(4),
-        waiting_fee: write_cost.saturating_mul(3),
-        waking_fee: write_cost.saturating_mul(2),
-        reservation_fee: write_cost.saturating_mul(2),
+        sending_fee: costs.write.calc(2.into()),
```

Comment:
another renaming suggestion
```suggestion
        sending_fee: costs.write.mult(2.into()),
```

Reply:
So, you wanna `mult` or `calc_for` ? Actually I also don't like `calc`, but cannot come up with another name.

Reply:
Sorry, but I chose `cost_for` and `cost_for_one` in order to be able to add in future `cost_for_with_bytes`